setup.cfg

```
@@ -1,5 +1,5 @@
 [metadata]
-description-file = README.rst
+description_file = README.rst
```

This isn't actually valid/used, it turns out. It should actually be:
```suggestion
long_description = file: README.rst
```
(Reference https://setuptools.pypa.io/en/latest/userguide/declarative_config.html)